torchhd/utils.py

```diff
@@ -0,0 +1,63 @@
+import torchhd.functional as functional
+
+def plot_basis_set(A, ax=None, **kwargs):
```

Comment:
Can you change the argument to something more descriptive? Maybe `memory` instead of `A`? And add typing information for the arguments and the return of the function?